packages/mobile/src/components/icon/arrow.tsx

```diff
@@ -58,3 +58,37 @@ export const DoubleRightArrowIcon: FunctionComponent<{
     </Svg>
   );
 };
+
+export const UpArrowIcon: FunctionComponent<{
+  color: string;
+  size: number;
+}> = ({ color, size }) => {
+  return (
+    <Svg width={size} height={size} viewBox="0 0 16 16" fill="none">
+      <Path
+        d="M3.5 10.25L8 5.75L12.5 10.25"
+        stroke={color}
+        stroke-width="2"
+        stroke-linecap="round"
+        stroke-linejoin="round"
```

Comment:
```suggestion
        stroke={color}
        strokeWidth="2"
        strokeLinecap="round"
        strokeLinejoin="round"
```
think you should make transformations as React attributes